packages/core/styles.css

```diff
@@ -12,3 +12,13 @@
   position: relative;
   z-index: 0;
 }
+
+[data-resize-overlay] {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 9999;
+  cursor: col-resize;
+}
```

Comment:
This file has moved so now facing a conflict. Can you move to a `styles.css` file under the component it's used and import it into the component like `import "./styles.css"`?